src/runtime/mod.rs

```diff
@@ -13,7 +13,9 @@ thread_local! {
     pub(crate) static CONTEXT: RuntimeContext = RuntimeContext::new();
 }
 
-/// The Runtime executor
+/// The Runtime Executor
```

Comment:
Add a whitespace line in-between. Rustdoc relies on this line for how it formats the docs.